corehq/apps/sms/templates/sms/whatsapp_templates.html

```diff
@@ -0,0 +1,49 @@
+{% extends "hqwebapp/base_section.html" %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% block title %}
+  {% trans "WhatsApp Templates" %}
+{% endblock %}
+
+
+{% block page_content %}
+  <h2>{% trans "WhatsApp Templates" %}</h2>
+  <hr/>
+  <p>
+    {% blocktrans %}
+      To send a template message, copy and paste the <strong>Template String</strong> as the message text, replacing each <code>{var}</code> with the variables you want to add in to the template. Don't forget the <code>~</code> at the end.
+    {% endblocktrans %}
+  </p>
+  <table class="table table-bordered table-striped">
+    <thead>
+      <tr>
+        <th>{% trans "Template Name" %}</th>
+        <th>{% trans "Body" %}</th>
+        <th>{% trans "Category" %}</th>
+        <th>{% trans "Status" %}</th>
+        <th>{% trans "Language" %}</th>
+        <th>{% trans "Template String" %}</th>
+      </tr>
+    </thead>
+    <tbody>
+      {% for template in wa_templates %}
+        <tr>
+          <td>{{ template.name }}</td>
+          <td>
+            {% for component in template.components %}
+              {% if component.type == 'BODY' %}
+                {{ component.text }}
+              {% endif %}
+            {% endfor %}
+          </td>
+          <td>{{ template.category }}</td>
+          <td>{{ template.status }}</td>
+          <td>{{ template.language }}</td>
+          <td>{{ template.template_string }}</td>
```

Comment:
It could be convenient to have a "Copy to clipboard" button, either here or in its own column.